fidesctl/src/fidesctl/core/annotate_dataset.py

```diff
@@ -103,69 +101,68 @@ def annotate_dataset(
         Write the amended dataset file in place
     """
     output_dataset = []
+    url = config.cli.server_url
+    headers = config.user.request_headers
 
     # Make the user aware of the data_categories visualizer
     click.secho(
-        f"""For reference, open the data category visualizer at either (localhost if running container):
-        {visualize.get_visualize_url(resource_type, "graphs")}
-        {visualize.get_visualize_url(resource_type, "text")}
-    """,
+        "For reference, you can use the Taxonomy explorer at the '/<resource_type>/visalize/graphs/' endpoint on a running fidesctl webserver.",
```

Comment:
Copy changes aside, I want to call out the typo fix: `visalize` --> `visualize`

```suggestion
        "View the entire fideslang taxonomy using the Taxonomy explorer! While running the fidesctl server, visit:\n\t/<resource_type>/visualize/graphs/",
```

Also, `get_visualize_url` in `core/visualize.py` is now completely unused, so let's remove it.

Reply:
done